app.rb

```diff
@@ -61,12 +61,19 @@ class MakersBNB < Sinatra::Base
 
   get '/spaces/bookings/:id' do
     id = params[:id]
+    p id
```

Comment:
Does this still need to print at the commit stage?

Reply:
Will change before we merge.